### ISSUE TITLE
Tasks plugin recommendations

### ISSUE THREAD

**Issue by xxxuanxx** [msgbyte/tailchat#254]
Can the Tasks plugin add the function of record modification and deletion?

**Comment by moonrailgun** [comment]
i remember we can click completed.

**Comment by xxxuanxx** [comment]
Yes, but once it is submitted, it cannot be modified. Even if you click Finish. The wrong record cannot be deleted, and you may be confused when you check it again later.

**Comment by moonrailgun** [comment]
well, no time to support this feature, waiting a good man and welcome PR
